jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -403,6 +417,7 @@ def multipass_pipeline(jobs) {
                     SCRIPT_DIR,
                     server_era)
             step030_customize_settings(SCRIPT_DIR, job['puppet_settings'])
+            step035_customize_hocon_settings(SCRIPT_DIR, job['hocon_settings'])
```

Comment:
Yeah, that's true.  It would mean that we'd might have to have conditional logic in the test setup that's different for OSS vs. PE for particular settings whereas if we moved when we run the step down to a later time - after the agent on the master has been disabled - we could eliminate the conditional setup.  That said, I don't have a strong opinion about this.  If we think this is ultimately the best spot for this step, I'm fine with leaving it how it is here.